web/sidebars.js

```diff
@@ -37,6 +38,19 @@ module.exports = {
             'tutorials/todo-app/dependencies',
             'tutorials/todo-app/the-end'
           ]
+        },
+        {
+          type: 'category',
+          label: 'Dev Excuses app',
+          collapsed: true,
+          items: [
+            'tutorials/dev-excuses-app',
+            'tutorials/dev-excuses-app/creating-the-project',
```

Comment:
Idea: maybe you could add `0n_` in front of each file name, to make it easier when reviewing the code / working on it to figure out the order? So `01-creating-the-project`. That way they will be ordered in github when doing PR, and also in local editor when you open the source tree.